### PR TITLE
Use content store for policy links instead of linkables

### PR DESCRIPTION
**Pull request by kevindew**
This won't build green until https://github.com/alphagov/govuk-content-schemas/pull/563 is merged in.

This is a system that replaces Whitehall using the Publishing API's linkables endpoint to render frontend content. It works by publishing the featured policies as links to the organisations placeholder content item). Then when rendering a page it'll access the content item.

For a painless transition this falls back to linkables if there aren't links in the content item. Once we have republished everything we could remove the fall back.

### CODE REVIEW

File: app/controllers/organisations_controller.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class OrganisationsController < PublicFacingController
   include CacheControlHelper
 
+  CONTENT_NOT_FOUND = Class.new
```
Comment (kevindew) [inline app/controllers/organisations_controller.rb]: It's used as a means to memoize a value that might not exist: https://github.com/alphagov/whitehall/blob/c3c2aa5ddce1521ec48c24a531522f436c043ad2/app/controllers/organisations_controller.rb#L141-L148 as it returning nil won't allow memoize. We had similar confusion when we used this pattern on Pub API too so do wonder if there is a more idiomatic ruby pattern we can use 🤔 - any different approaches to solving this issue in Whitehall that are more consistent?

**Comment by kevindew** [comment]
Thanks for the review @steventux

File: app/controllers/organisations_controller.rb (hunk 1 above)
Comment (steventux) [inline app/controllers/organisations_controller.rb]: Ah I didn't think about the memoization of the rescue block return value. Makes more sense now.
Perhaps for the sake of clarity a custom class with a descriptive name could be used?
`MissingContentItem`, `NonExistentContentItem` or  something similar (admittedly neither of these suggestions really stands out).

**Comment by kevindew** [comment]
Thanks @bevanloon 

I've updated the approach to memoizing the organisation content, what do you guys think?

**Pull request merged**